ujin/programmers/day06/예산.js

```diff
@@ -0,0 +1,11 @@
+function solution(d, budget) {
+  d.sort((a, b) => a - b);
+  let answer = 0;
+  for (let i = 0; answer <= budget; i++) {
+    if (answer + d[i] <= budget) {
+      answer += d[i];
```

Comment:
매개변수 (`d`) 를 정렬한 값을 새로운 변수에 담아서 사용하는 것이 좋을 것 같습니다.
```suggestion
const sortedNumbers = d.sort((a, b) => a - b);
  let answer = 0;
  for (let i = 0; answer <= budget; i++) {
    if (answer + sortedNumbers[i] <= budget) {
      answer += sortedNumbers[i];
```